lib/bamboozled/api/employee.rb

```diff
@@ -61,6 +61,13 @@ def add(employee_details)
         request(:post, "employees/", options)
       end
 
+      def update(bamboo_id: nil, employee_details: nil)
```

Comment:
I think it might be better to use standard parameters here, at least until we are ready to stop supporting Ruby 2.0, since both `bamboo_id` and `employee_details` are required. The way it is right now allows me to call `update` without params.